dll/cpl/desk/lang/en-US.rc

```diff
@@ -179,14 +179,14 @@ CAPTION "General"
 FONT 8, "MS Shell Dlg"
 BEGIN
     GROUPBOX "Display", -1, 7, 6, 237, 56, WS_GROUP
-    LTEXT "Font Size:", -1, 14, 20, 222, 9
+    LTEXT "Font Size:", -1, 14, 20, 222, 9//FIXME: add accel
```

Comment:
.... and that? @Doug-Lyons

Reply:
[We already posted that one](https://github.com/reactos/reactos/pull/7298#discussion_r1741215836), it's called "&DPI setting:".

"&Font Size:" is from 2000:
![2000](https://github.com/user-attachments/assets/4e141ad1-3776-4f1e-8f4f-c3147dc8a993)

Reply:
Again with a full screen shot.

![W2K3SP2_Default_Monitor](https://github.com/user-attachments/assets/c3a65372-d4ee-4b6f-83ff-c7dc65fa5656)

Reply:
Resolved. I applied the new thing for english and german (some others had the new one already). For all others I added a comment requesting the native speakers do update their translation.